lib/mocks/mockStore.js

```diff
@@ -0,0 +1,49 @@
+var sinon = require('sinon');
```

Comment:
IMO files dedicated to testing should still go under `/test/` and be required from the `-spec.js` files. Just so we don't have everything under `/lib`

@nialldonnellyfh WDYT?